documentation/docs/05-load.md

```diff
@@ -335,37 +434,93 @@ If an _unexpected_ error is thrown, SvelteKit will invoke [`handleError`](/docs/
 
 To redirect users, use the `redirect` helper from `@sveltejs/kit` to specify the location to which they should be redirected alongside a `3xx` status code.
 
-```diff
-/// file: src/routes/admin/+layout.server.js
--import { error } from '@sveltejs/kit';
-+import { error, redirect } from '@sveltejs/kit';
+```js
+/// file: src/routes/user/+layout.server.js
+// @filename: ambient.d.ts
+declare namespace App {
+	interface Locals {
+		user?: {
+			name: string;
+		}
+	}
+}
+
+// @filename: index.js
+// ---cut---
+import { redirect } from '@sveltejs/kit';
 
 /** @type {import('./$types').LayoutServerLoad} */
 export function load({ locals }) {
 	if (!locals.user) {
--		throw error(401, 'not logged in');
-+		throw redirect(307, '/login');
-	}
-
-	if (!locals.user.isAdmin) {
-		throw error(403, 'not an admin');
+		throw redirect(307, '/login');
 	}
 }
 ```
 
 ### Invalidation
 
-SvelteKit tracks the dependencies of each `load` function to avoid re-running it unnecessarily during navigation. For example, a `load` function in a root `+layout.js` doesn't need to re-run when you navigate from one page to another unless it references `url` or a member of `params` that changed since the last navigation.
+SvelteKit tracks the dependencies of each `load` function to avoid re-running it unnecessarily during navigation. For example, a `load` function in a root `+layout.js` doesn't need to re-run when you navigate from one page to another unless it references `url` or a member of `params` that changed since the last navigation. `load` functions also take into account parent `load` functions, if you reference them through `await parent()` - in this case, if an upper `load` function is rerun, so is the `load` function that does `await parent()`.
+
+`load` functions do not only run when you navigate, they can also be triggered to rerun by `invalidate` or `invalidateAll` (which we get to in the next paragraph). `invalidate` is closely connected to `fetch` and `depends`, which are both methods passed to `load`. Making a `fetch` request automatically registers the fetched URL as a dependency of the load function. `depends` does the same in a manual way and registers the specified URL as a dependency. You can then use `invalidate` and pass one of the registered URLs (or a function, if you need to decide based on a pattern rather than the full URL) to it to rerun the `load` function:
```

Comment:
About `depends`. It took a while this morning for me to figure out that you call `depends` in the `load` _you are depending on_, not elsewhere (like a `load` further down the tree.) Maybe this was just because it was my first time using it. I'm fine with the word `depends` now, but maybe we can point out that it means (AFIACT) "define a cache key for this `load` function."